sim/shaman/talents.go

```diff
@@ -650,6 +651,7 @@ func (shaman *Shaman) applySearingFlames() {
 
 				// recalc damage based on stacks, testing with searing totem seems to indicate the damage is updated dynamically on refesh
 				// instantly taking the bonus of any procs or buffs and applying it times the number of stacks
+				dot.SnapshotCritChance = spell.CritMultiplier
```

Comment:
This should make it crit instead of the spell calc as it deals 0 damage. The dot part was the important one. That being said. This should be set to the crit chance not the crit multiplier (how high the crit damage is)